### DISCUSSION TITLE
Factory and ERC721 token contract

### DISCUSSION THREAD

**Discussion by damianlluch** [hashgraph/hedera-smart-contracts#28] [Q&A]
Hello, in my company we are investigating how to add Hedera to our marketplace and I have doubts on how to implement it.
In my first place we have a factory contract, and on the other hand the token contract, an ERC721.
Is there any example of how to do this? I have tried using the SDK with the account I created, but it tells me that the contract exceeds the allowed limit.
The Discord forum talks about splitting the contract into small chunks, but I haven't found an example of how to do it.
Any suggestions or examples?

thanks

**Comment by Nana-EC** [comment]
Hi @damianlluch 
I believe this question is related to #27 and my response there should be helpful.
Feel free to clarify here if you're still stuck
